features/bootstrap/SwaggerContext.php

```diff
@@ -0,0 +1,260 @@
+    }
+
+    /**
+     * @Then the Swagger class ":class" exist
+     */
+    public function assertTheSwaggerClassExist($className)
+    {
+        $this->getClassInfos($className);
+    }
+
+    /**
+     * @Then the Swagger class ":class" not exist
+     */
+    public function assertTheSwaggerClassNotExist($className)
+    {
+        try {
+            $this->getClassInfos($className);
+
+            throw new \PHPUnit_Framework_ExpectationFailedException(sprintf('The class "%s" exist.', $className));
+        } catch (\Exception $exception) {
+            // an exception must be catched
+        }
+    }
+
+    /**
+     * @Then the value of the node ":node" of the Swagger class ":class" is ":value"
+     */
+    public function assertNodeValueIs($nodeName, $className, $value)
+    {
+        $classInfos = $this->getClassInfos($className);
+        \PHPUnit_Framework_Assert::assertEquals($this->propertyAccessor->getValue($classInfos, $nodeName), $value);
+    }
+
+    /**
+     * @Then the value of the node ":node" of the property ":prop" of the Swagger class ":class" is ":value"
+     */
+    public function assertPropertyNodeValueIs($nodeName, $propertyName, $className, $value)
+    {
+        $property = $this->getProperty($propertyName, $className);
+
+        \PHPUnit_Framework_Assert::assertEquals($this->propertyAccessor->getValue($property, $nodeName), $value);
+    }
+
+    /**
+     * @Then the value of the node ":node" of the operation ":operation" of the Swagger class ":class" is ":value"
+     */
+    public function assertOperationNodeValueIs($nodeName, $operationMethod, $className, $value)
+    {
+        $property = $this->getOperation($operationMethod, $className);
+
+        \PHPUnit_Framework_Assert::assertEquals($this->propertyAccessor->getValue($property, $nodeName), $value);
+    }
+
+    /**
+     * @Then :nb operations are available for Swagger class ":class"
+     */
+    public function assertNbOperationsExist($nb, $className)
+    {
+        $operations = $this->getOperations($className);
+
+        \PHPUnit_Framework_Assert::assertEquals($nb, count($operations));
+    }
+
+    /**
+     * @Then :nb properties are available for Swagger class ":class"
+     */
+    public function assertNbPropertiesExist($nb, $className)
+    {
+        $properties = $this->getProperties($className);
+
+        \PHPUnit_Framework_Assert::assertEquals($nb, count($properties));
+    }
+
+    /**
+     * @Then ":prop" property doesn't exist for the Swagger class ":class"
+     */
+    public function assertPropertyNotExist($propertyName, $className)
+    {
+        try {
+            $this->getProperty($propertyName, $className);
+
+            throw new \PHPUnit_Framework_ExpectationFailedException(
+                sprintf('The property "%s" for the class "%s" exist.', $propertyName, $className)
+            );
+        } catch (\Exception $exception) {
+            // an exception must be catched
```

Comment:
I guess it's to respect PSR-2 that forbid empty catch without comment. Maybe a better message can be found :-)